api/routes/database.py

```
@@ -36,27 +40,38 @@
         # Check for MySQL URL
         elif url.startswith("mysql://"):
             try:
                 # Attempt to connect/load using the MySQL loader
                 success, result = MySQLLoader.load(request.state.user_id, url)
             except (ValueError, ConnectionError) as e:
                 logging.error("MySQL connection error: %s", str(e))
-                raise HTTPException(status_code=500, detail="Failed to connect to MySQL database")
-        
+                raise HTTPException(
+                    status_code=500, detail="Failed to connect to MySQL database"
+                )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)